.github/workflows/run-cms-regression.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           npm ci
           ### Run Cypress CMS tests
       - name: Run tests
-        run: CYPRESS_baseUrl=${{ inputs.base_url }} CYPRESS_front_end_base_url=${{ inputs.front_end_base_url }} CYPRESS_admin_password='${{ secrets.admin_password }}' npm run cypress:cms
+        run: CYPRESS_baseUrl=${{ inputs.base_url }} CYPRESS_front_end_base_url=${{ inputs.front_end_base_url }} CYPRESS_admin_password='${{ secrets.admin_password }}' npm run cypress:cms-parallel
```

Comment:
I will change this and the ode file before I am done with the pr. It's only been changed to allow me to test on github actions rather than locally

Reply:
Never mind!